presto-main/src/test/java/io/prestosql/sql/planner/TestPredicatePushdown.java

```diff
@@ -520,9 +522,68 @@ public void testTablePredicateIsExtracted()
                                                 "orders",
                                                 ImmutableMap.of("ORDERSTATUS", "orderstatus"))),
                                 anyTree(
-                                        filter("CAST(NAME AS varchar(1)) IN ('F', 'O')",
+                                        filter(
+                                                "CAST(NAME AS varchar(1)) IN ('F', 'O') AND (NOT (CAST(NAME AS VARCHAR(1))) IS NULL)",
```

Comment:
Add ``// TODO we should eliminate NULL check on `CAST(NAME AS VARCHAR(1))` since it's either 'F' or 'O'``